models/repeater.js

```diff
@@ -77,6 +80,107 @@ repeaterSchema.index({
   },
 });
 
+// Hooks for device traps notifications
+repeaterSchema.pre('save', function(callback) {
+  let repeater = this;
+  let changedAttrs = {};
+  let requestOptions = {};
+  let attrsList = repeater.modifiedPaths();
+
+  if (attrsList.length > 0) {
+    // Send modified fields if callback exists
+    Config.findOne({is_default: true},
+                  {traps_callbacks: true}).lean()
+    .exec(function(err, defConfig) {
+      if (err || !defConfig || !defConfig.traps_callbacks ||
+                !defConfig.traps_callbacks.repeaters_crud) {
+        return callback(err);
+      }
+      const promises =
+        defConfig.traps_callbacks.repeaters_crud.map((repeaterCrud) => {
+          let callbackUrl = repeaterCrud.url;
+          let callbackAuthUser = repeaterCrud.user;
+          let callbackAuthSecret = repeaterCrud.secret;
+          if (callbackUrl) {
+            attrsList.forEach((attr) => {
+              changedAttrs[attr] = repeater[attr];
+            });
+            // Nothing to send - don't call trap
+            if (Object.keys(attrsList).length === 0) {
+              return Promise.resolve();
+            }
+            requestOptions.url = callbackUrl;
+            requestOptions.method = 'PUT';
+            requestOptions.json = {
+              'id': repeater._id,
+              'type': 'repeater',
+              'changes': attrsList,
```

Comment:
Tem que mandar as alterações, não a lista de atributos
```suggestion
              'changes': changedAttrs,
```